### PR TITLE
Bump version to 0.8.0.dev2

### PR DESCRIPTION
**Pull request by yosukehigashi**
Bumps the version to 0.8.0.dev2. This is a pre-release, so pip will ignore it by default (https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions).

### CODE REVIEW

**Comment by liwii** [comment]
I'll wait until #123 is merged!

**Comment by yosukehigashi** [comment]
Just merged!

**Review by liwii — APPROVED** [review]

**Pull request merged**
